### PR TITLE
fix(node-sdk): Properly calculate length of utf-8 inputs

### PR DESCRIPTION
**Pull request by bhelx**
Properly calculate the length of utf-8 inputs. I'll work on getting unicode chars into our other tests as well and see if we need to fix any other SDKs

### CODE REVIEW

File: node/src/index.ts [hunk 1]
```diff
@@ -94,6 +94,12 @@ if (process.env.EXTISM_PATH) {
 
 const lib = locate(searchPath);
 
+// get the true byte size of a utf-8 string
+const byteSize = (data: string | Buffer) => {
```
Comment (bhelx) [inline node/src/index.ts:98]: perhaps instead of doing this we can just always turn the string into a buffer. There might be a better way to do this.

Comment (zshipko) [reply node/src/index.ts:98]: It looks like there is also `Buffer.byteLength`: https://nodejs.org/api/buffer.html#static-method-bufferbytelengthstring-encoding

Comment (bhelx) [reply node/src/index.ts:98]: Thanks i'll try that

Comment (bhelx) [reply node/src/index.ts]: Seems to work!

**Review by nilslice — APPROVED** [review]

**Pull request merged**